### ISSUE TITLE
Multiple sites showing no controls on settings screen after update to v3.0.3 

### ISSUE THREAD

**Issue by jb510** [cloudflare/CloudFlare-Wordpress#85]
I've now seen this on several sites (across multiple CF accounts). After updating from V1 to V3 I go to the settings screen and it displays the Active Zone, the 3 blue nav buttons, but none of the control modules.  

https://cloudup.com/ceTUmHLvOFT

This is from my personal blog. WP Engine hosted, PHP 5.6.32, Mac Chrome Version 50.0.2661.75.

It's worth noting I _have_ seen the control on some sites, although when I do see the control them they don't work (CSRF Error per issue #82). Hence, I don't think it's a browser issue. Further I've cleared site cache / tried in incognito and with FireFox on the sites they don't show, they don't show in all those test cases.

**Comment by thellimist** [comment]
This is also a known issue. We're working to fix this issue. When 

Thanks.

**Comment by jwineman** [comment]
Lets leave this open until the fix is released.  Comment the fix and close the issue.

**Comment by XhmikosR** [comment]
This is the same issue I describe in my #60 issue.

**Comment by jwineman** [comment]
@XhmikosR @jb510 Is this fixed in 3.0.4 for you?

**Comment by jb510** [comment]
@jwineman I can now see the react elements on that page, so yes #85 seems fixed in 3.0.4 (I'll test more sites where I saw this behavior later).  

However, the CSRF issue with InfiniteWP persists. FYI, I've bugged IWP support and they're working on it. If there still isn't an update in a week I'll bug Vinodh directly.  I seem to recall there are some tricky issues with stream_copy_to_stream() and rewind()...  but trusting they'll sort it out.
